server/src/pgboss-worker.ts

```
@@ -358,6 +363,78 @@ async function processRunExecution(job: Job<ExecuteRunData>) {
+        log: 'Aborted: Browser not found or already closed'
+      });
+      
+      logger.log('warn', `Browser not found for run ${runId}`);
+      return true;
+    }
+
+    const currentLog = browser.interpreter.debugMessages.join('\n');
+    const serializableOutput = browser.interpreter.serializableData.reduce((reducedObject, item, index) => {
+      return {
+        [`item-${index}`]: item,
+        ...reducedObject,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)